### ISSUE TITLE
İos video quality change problem 

### ISSUE THREAD

**Issue by kaganzdmr58** [jwplayer/jwplayer#4019]
I am developing a mobile application for ios and android devices, video quality can be adjusted in android, but it seems that the video quality is set as default in ios. I want it to give the possibility to change it manually in iOS, how can I do that?

**Comment by madhekar** [comment]
Use the preferredPeakBitRate property on your playeritem you need to pass a
valid bandwidth value.

**Comment by kaganzdmr58** [comment]
In android, the user can set the quality, but in ios, the user cannot set it because it cannot be clicked.
Please look at the pictures.

![android_quality_1](https://user-images.githubusercontent.com/58965229/234512505-27e2f171-fb56-4fec-b75c-2e8e39bb6db8.png)
![android_quality2](https://user-images.githubusercontent.com/58965229/234512528-2d870db7-bb46-487c-b484-da71dd94f8e7.png)
![Simulator Screen Shot - iPhone 14 - 2023-04-26 at 11 08 52](https://user-images.githubusercontent.com/58965229/234512535-8a23820d-81f1-400a-8bd8-242e83d3b1ab.png)

**Comment by kaganzdmr58** [comment]
Hi All, here I see the problem is bigger, when I wanted to test from JWPlayer official page I got different results for safari and chrome. When I wanted to make m3u8 video on safari, I saw that the quality part was turned off and m3u8 video quality can be changed via chrome. Here the problem persists not only in phones but also in web browsers. I also confirmed that the problem is not the video I produced, because the video presented as the JWplayer test video has the same problem.

jwplayer stream test url  ->  https://developer-tools.jwplayer.com/stream-tester
jwplayer example video  ->  https://content.jwplatform.com/manifests/yp34SRmf.m3u8 



![chrome](https://user-images.githubusercontent.com/58965229/235428821-0357eee1-b6e8-4739-b886-0606c4fe0cbb.png)
![safari](https://user-images.githubusercontent.com/58965229/235428838-8a6f1d51-5336-42be-9dea-cd8981d0a0cc.png)
